light-zk.js/src/utxo.ts

```diff
@@ -390,7 +419,7 @@ export class Utxo {
         ).fill(0),
       ]);
       includeAppData = false;
-
+      // @matteo: why not performing the check below before the array initialization above? we can save some time/space
```

Comment:
didn't we clarify this?

Reply:
oh I thought that was removed alongside other ```@matteo``` tagged comments but in the rebase they reappeared. let me clean them up